Client/Frontend/Browser/Playlist/Cells/PlaylistCellRedacted.swift

```diff
@@ -0,0 +1,151 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import CoreData
+import Shared
+import BraveUI
+import DesignSystem
+import BraveShared
+import Data
+
+private struct PlaylistRedactedHeaderView: View {
+  var title: String?
+  var creatorName: String?
+  
+  var body: some View {
+    HStack {
+      VStack(alignment: .leading) {
+        Text(title ?? "PlaylistTitlePlaceholder")
+          .font(.title3.weight(.medium))
+          .foregroundColor(Color(.bravePrimary))
+          .multilineTextAlignment(.leading)
+          .redacted(reason: title == nil ? .placeholder : [])
+          .shimmer(title == nil)
+        
+        Text(creatorName ?? "CreatorName")
+          .font(.footnote)
+          .foregroundColor(Color(.braveLabel))
+          .multilineTextAlignment(.leading)
+          .redacted(reason: creatorName == nil ? .placeholder : [])
+          .shimmer(creatorName == nil)
+      }
+      .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .leading)
+      
+      Button("+ Add", action: {})
```

Comment:
Still a problem: This should not be a `+` unicode character, use `Label` & the `plus` system image

Reply:
Why? This is a `placeholder` view. It's always `redacted` and `shimmer(true)`
I will change it anyway, but this isn't a problem.

Reply:
I mean that the UIKit version should also be fixed, though during my last test I didn't see a `+` character at all

Reply:
Fixed